run-tests

```diff
@@ -1,20 +1,30 @@
 #!/bin/bash -e
 
 case $0 in (/*) cd "${0%/*}/";; (*/*) cd "./${0%/*}";; esac
+if [[ "$#" -eq 0 ]]; then
+    set -- qrexec/tests
+else
+    for i; do
+        case $i in
+        (qrexec/tests|qrexec/tests/*) :;;
+        (*) printf 'Unsupported path %q\n' "$i" >&2; exit 1;;
```

Comment:
Please don't. This breaks using any options for pytest via the script.

Reply:
I guess I wasn't clear enough. Do not add arguments validation to this script. This still breaks several cases (like options with arguments). And I forbid you to re-implement pytest arguments parser here.